### PR TITLE
[eth] Improve docs and repo

### PR DESCRIPTION
**Pull request by ali-bahjati**
This PR:
- Updates Deploying.md doc according to the feedbacks that I received.
- Removes `.openzeppelin` directory and adds it to gitignore as we are no longer using truffle for upgrades
  and it is not needed anymore.
- Adds `*mnemonic*` to gitignore to avoid mistakes.

### CODE REVIEW

File: ethereum/Deploying.md [hunk 1]
```diff
@@ -1,61 +1,109 @@
 # Deploying Contracts to Production
 
-Running the Truffle migrations in [`migrations/prod`](migrations/prod) or [`migrations/prod-receiver`](migrations/prod-receiver/) will deploy the contracts to production. The `prod-receiver` migrations should be used when you need to deploy to a chain that is unsupported by the Wormhole network. The Wormhole Receiver contract acts as a read-only Wormhole endpoint that can verify Wormhole messages even if the Wormhole network has not yet connected the chain.
+## EVM network configurations
```
Comment (jayantk) [inline ethereum/Deploying.md:3]: no plural on "configuration", also in the sentence below.

File: ethereum/Deploying.md [hunk 2]
```diff
@@ -1,61 +1,109 @@
...
+
+Each network that Pyth is deployed on has some configurations stored on this repo as described below:
+
+1. [`truffle-config.js`](./truffle-config.js) contains list of different networks with their configurations that includes:
+   - `provider`: It is the network provider which is an HDWalletProvider with mnemonic stored on $MNEMONIC
```
Comment (jayantk) [inline ethereum/Deploying.md:8]: suggest starting with "The network provider" . "it is" is redundant

"stored in the $MNEMONIC environment variable"

File: ethereum/Deploying.md [hunk 3]
```diff
@@ -1,61 +1,109 @@
...
-As a sanity check, it is recommended to deploy the migrations in `migrations/prod` to the Truffle `development` network first. You can do this by using the configuration values in [`.env.prod.development`](.env.prod.development).
-
-As a result of this process for some files (with the network id in their name) in `networks` and `.openzeppelin` directory might change which need to be committed (if they are result of a production deployment).
-
-If you are deploying to a new network, please add the new contract address to consumer facing libraries and documentations.
-
-To do so, add the contract address to both [Pyth Gitbook EVM Page](https://github.com/pyth-network/pyth-gitbook/blob/main/consumers/evm.md) and [pyth-evm-js package](https://github.com/pyth-network/pyth-js/blob/main/pyth-evm-js/src/index.ts#L13). You also need to add the new network address to [pyth-evm-js relaying example](https://github.com/pyth-network/pyth-js/blob/main/pyth-evm-js/src/examples/EvmRelay.ts#L47).
-
-## `networks` directory
+1. Follow the installation instructions on [README.md](./README.md).
```
Comment (jayantk) [inline ethereum/Deploying.md:51]: in the README

**Review by jayantk — APPROVED** [review]
nice! left a couple minor writing comments

File: ethereum/Deploying.md [hunk 4]
```diff
@@ -104,10 +152,11 @@ It will create a new file `PythUpgradable_merged.sol` which you can use in the e
 
 # Troubleshooting
 
+- You get `digital envelope routines::unsupported` error. The current truffle version is old and if you are using a new Node version you might need
```
Comment (cctdaniel) [inline ethereum/Deploying.md:155]: If you get `digital envelope routines::unsupported` error. it means that the current truffle version is old...

**Review by cctdaniel — APPROVED** [review]

**Pull request merged**